tests/file-magic-keywords/test.rules

```diff
@@ -0,0 +1 @@
+alert http any any -> any any (msg:"store png images"; filemagic:"PNG image data"; filestore; sid:1; rev:1;)
```

Comment:
also, since we're using filestore feat, perhaps a test for whether the file was stored or not would be nice too